src/clj/rems/application/commands.clj

```diff
@@ -268,7 +268,7 @@
 
 (defn already-member-error [application userid]
   (when (contains? (all-members application) userid)
-    {:errors [{:type :already-member :application-id (:id application)}]}))
+    {:errors [{:type :already-member :application-id (:application/id application)}]}))
```

Comment:
There seems to be a mismatch as it's :application/id here but the handler in ::accept-invitation accesses :application-id.

Reply:
I see only `command-handler :application.command/accept-invitation` accessing `(:application-id cmd)`, not the application model.